recipes/ncurses/ncurses.inc

```diff
@@ -91,7 +91,7 @@ AUTO_PACKAGE_UTILS_RDEPENDS += "libncurses"
 FILES_${PN} = "${datadir}/tabset ${sysconfdir}/terminfo"
 RDEPENDS_${PN}-tools += "${AUTO_PACKAGE_UTILS_PROVIDES}"
 FILES_${PN}-terminfo = "${datadir}/terminfo ${libdir}/terminfo"
-FILES_${PN}-dev += "${bindir}/ncurses5-config"
+FILES_${PN}-dev += "${bindir}/ncurses?-config"
```

Comment:
probably safer with [5-6] if you want to do pattern matching, to avoid random explodes when they make a ncurses7